pkg/logger/context.go

```diff
@@ -28,33 +28,33 @@ type Context interface {
 	// Append... methods
 	WriteOut(level Level, message string)
 
-	// AppendScope adds a scope value to this context.
+	// SetScope adds a scope value to this context.
```

Comment:
nit: Not really adding to the context, as the comment says
Same for SetCaller and SetError

```suggestion
	// SetScope sets the scope value for this context.
```